### ISSUE TITLE
OS X compilation still fails for branches/release-v3.2.0. w/ no-werror flag – x86

### ISSUE THREAD

**Issue by haf** [EventStore/EventStore#595]
```
$  ./build.sh full 3.1.1 x86 release no-werror

10 minutes later into my day...

1 warning generated.
  LIBTOOL-STATIC /Users/h/dev/logibit/_scratch/EventStore/v8/out/ia32.release/obj.host/third_party/icu/libicuuc.a
clang: error: no such file or directory: '/Users/h/dev/logibit/_scratch/EventStore/v8/out/ia32.release/libv8_base.x64.a'
clang: error: no such file or directory: '/Users/h/dev/logibit/_scratch/EventStore/v8/out/ia32.release/libv8_nosnapshot.x64.a'
```

**Comment by pgermishuys** [comment]
Will take a look at this now.

**Comment by haf** [comment]
My guess is that it's not picking up the x86 flag everywhere.

**Comment by pgermishuys** [comment]
Right, just saw that now, x64 is what you need to pass there.

**Comment by haf** [comment]
Docs say I can choose.

**Comment by pgermishuys** [comment]
The docs will need updating. Thanks @haf

**Comment by haf** [comment]
OK, so running ES on x86 is absolutely not supported?

**Comment by pgermishuys** [comment]
@haf see updated response.

**Comment by pgermishuys** [comment]
@haf unfortunately regardless of whether this was working before, the build scripts are littered with hardcoded "x64".
example: 
https://github.com/EventStore/EventStore/blob/release-v3.2.0/build.sh#L296-L297
https://github.com/EventStore/EventStore/blob/release-v3.2.0/build.sh#L301

so unfortunately it won't work for x86 right now.

**Comment by haf** [comment]
Great, so then my commit message and change is accurate.

**Comment by pgermishuys** [comment]
Yes, will review and let you know soon.
Thank you for the feedback.

**Comment by jen20** [comment]
This was resolved differently - we don't support x86 builds at all anymore via our build infrastructure.

**Comment by gregoryyoung** [comment]
@haf what is the benefit you are seeing in x86 builds?

On Mon, Sep 7, 2015 at 11:10 PM, James Nugent <notifications@github.com>
wrote:

> Closed #595 <https://github.com/EventStore/EventStore/issues/595>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/EventStore/EventStore/issues/595#event-403049099>.
>



-- 
Studying for the Turing test

**Comment by haf** [comment]
@gregoryyoung  Supporting the default OS X mono installation mostly, part lower memory footprint.
